packages/kit/src/runtime/server/page/actions.js

```diff
@@ -24,8 +24,7 @@ export async function handle_action_json_request(event, options, server) {
 	const actions = server?.actions;
 
 	if (!actions) {
-		// TODO should this be a different error altogether?
-		const no_actions_error = new HttpError(
```

Comment:
Side note: This shows we weren't 100% adhereing to the "no `error` functions in our code" rule for a long time, the recent PRs only made it so it's done in a few more places.